.github/workflows/consistency-check.yaml

```
@@ -4,8 +4,9 @@ on:
   push:
     branches:
     - master
+    - release-1.3
```

```suggestion
    - release-*
```

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet